e2e/cypress/support/api/setup.js

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+Cypress.Commands.add('apiInitSetup', (options = {}) => {
+    const loginAfter = options.loginAfter || false;
```

Comment:
Yeah, that works but it will end up with required (ugly) curly braces e.g. `cy.apiInitSetup({})` when I don't need to pass `loginAfter`. Let me know if there's a way to make it work as `cy.apiInitSetup()`.

Reply:
I think you can default to just the empty object, i.e.:
```suggestion
Cypress.Commands.add('apiInitSetup', ({loginAfter = false} = {}) => {
```

Reply:
Done, see https://github.com/mattermost/mattermost-webapp/pull/5723/commits/1001ee2d68729cd38c63fb617ae5d78dbdceb959